public/js/index.js

```diff
@@ -1,5 +1,32 @@
-function displayMessage() {
-  return 'Hello, World!';
+function buildProduct(product) {
```

Comment:
Following the rules of Clean Code, by Robert C. Martin (uncle Bob), this method should be placed undernearth the method that calls it (buildProducts).